lib/tests/streamlit/elements/number_input_test.py

```diff
@@ -71,6 +72,13 @@ def test_just_disabled(self):
         c = self.get_delta_from_queue().new_element.number_input
         self.assertEqual(c.disabled, True)
 
+    def test_placeholder(self):
+        """Test that it can be called with placeholder param."""
+        st.number_input("the label", placeholder="Type a number...")
+
+        c = self.get_delta_from_queue().new_element.number_input
+        self.assertEqual(c.placeholder, "Type a number...")
+
```

Comment:
nit: another test could check that value is string "min" if not specified.

Reply:
`min` itself isn't sent to the frontend through proto as string, but instead changed to a number value: meaning either `min_value` or `0`. And these two cases are covered by `test_default_value_when_min_is_passed` and `test_just_disabled`.